src/common/valgrind_internal.h

```diff
@@ -59,6 +59,13 @@
 
 #define	VALGRIND_DO_COMMIT VALGRIND_PMC_DO_COMMIT
 
+#define VALGRIND_DO_PERSIST(addr, len) do {\
```

Comment:
TAB after #define, pls. Same in line #107.
(BTW, how could it pass 'make cstyle'?)